exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,6 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+
+var percentageOfStudents = 15 /23 * 100; 
+var percentageOfMentors = 8/23*100; 
+
```

Comment:
nice, you could use the `Math.round()` function to round the results to whole numbers